docs/product/metrics/index.mdx

```diff
@@ -47,21 +47,23 @@ Sentry incorporates protection mechanisms that will drop metrics if their cardin
 
 Sentry allows the use of specific characters for the components of a metric:
 
-- **Metric Name**: Supports all word characters including `_`, `-`, `.`. Any unsupported characters will be replaced by `_`.
+- **Metric Name**: All word characters including `_`, `-`, `.`. Any unsupported characters will be replaced by `_`.
 
-- **Metric Tag Keys**: Supports all word characters including `_`, `-`, `.`, `/`. Any unsupported characters will be removed from the tag key.
+- **Metric Unit**: ASCII alphanumeric characters, underscores, and digits.
 
-- **Metric Tag Values**: Supports all Unicode characters.
+- **Metric Tag Keys**: All word characters including `_`, `-`, `.`, `/`. Any unsupported characters will be removed from the tag key.
 
-- **Custom Metric Units**:  Custom units must consist of ASCII alphanumeric characters, underscores, and digits.
+- **Metric Tag Values**: All Unicode characters.
 
 More information about the normalization of unsupported characters can be found [here](https://develop.sentry.dev/sdk/metrics/#normalization).
 
-### Metric Tags Length
+### Maximum Lengths
 
-Metric tags have a maximum length:
-- Tag Keys: 32 characters.
-- Tag Values: 200 characters.
+Sentry sets a maximum length for the components of a metric:
+- **Metric Name**: 150 characters.
+- **Metric Unit**: 15 characters.
+- **Metric Tag Keys**: 32 characters.
+- **Metric Tag Values**: 200 characters.
```

Comment:
```suggestion
- **Metric Tag Key**: 32 characters.
- **Metric Tag Value**: 200 characters.
```

Reply:
Thanks! I was torn whether to use the plural or singular, will use your suggestions